### ISSUE TITLE
Fix USFIMR STAC asset projections

### ISSUE THREAD

**Issue by echeipesh** [azavea/noaa-flood-mapping#55]
In order to consume USFIMR items by Raster Vision we need to have them projected to GeoJSON in 4326.
Looks like there was an uncaught problem with the reprojection process.
See: https://github.com/azavea/noaa-flood-mapping/pull/37#discussion_r470629556

This issue is to resolve the error and update the catalog on `s3://usfimr-data`

**Comment by CloudNiner** [comment]
I asked Dan McGlone to take a look at this, and here's what he found:

1. It looks like the USFIMR_all shapefile has incorrect coordinate values

2. The individual shapefiles from https://sdml.ua.edu/usfimr/ plot correctly

3. He was able to generate a shapefile from these [regional KML files](https://www.google.com/maps/d/u/0/viewer?mid=1sEiTDPfPh-z4U8oe3PLWNvmwAzI&ll=38.91831065236737%2C-118.36207742195583&z=7) that also plot correctly in 4326. **However** this list of flood events is incomplete, accounting for only 38 of the 49 events in the USFIMR_all shapefile. 

Any preference on how to proceed? If we still want to use the USFIMR_all shapefile, we may need to contact the group that generated it for a corrected file or to have them regenerate it. We can also create a list of the individual shapefiles and write a script to download and combine them with something like [ogrmerge](https://gdal.org/programs/ogrmerge.html)

Attached here is the partially complete shapefile of events generated from the KML described in (3) 

[USFIMR_kml_merged.zip](https://github.com/azavea/noaa-flood-mapping/files/5076641/USFIMR_kml_merged.zip)

**Comment by echeipesh** [comment]
Looks like the issue is specific to the `USFIMR_all` shapefile. The individual shapefiles are good.
I've uploaded them to `s3://usfimr-data/shp/` and that has all 48 entries.

**Comment by CloudNiner** [comment]
Great. I can tweak the USFIMR script to pull all the individual shapefiles and use those. Will do so today.

**Comment by moradology** [comment]
Not sure how I missed this - reviewed one of the plotted events and it looked fine (looked like a plausible flood geometry). Perhaps I got lucky. The KML might be incomplete because USFIMR_all is *not* USFIMR only. As far as I can tell, it is the global FIMR dataset and it just happens to be mislabelled

**Comment by CloudNiner** [comment]
I ran this code to pull all the individual files and merge them:
```bash
aws s3 cp --recursive --quiet s3://usfimr-data/shp/ ./data/shapefiles
consolidated_file="./data/USFIMR_merged.shp"
for i in $(find ./data/shapefiles -name '*.shp'); do
    if [ ! -f "$consolidated_file" ]; then
        # first file - create the consolidated output file
        ogr2ogr -f "ESRI Shapefile" -t_srs "EPSG:4326" $consolidated_file $i
    else
        # update the output file with new file content
        ogr2ogr -f "ESRI Shapefile" -t_srs "EPSG:4326" -update -append $consolidated_file $i -nln "usfimr_merged"
    fi
done
```
The new merged file looks like this:
<img width="1139" alt="Screen Shot 2020-08-17 at 11 34 23 AM" src="https://user-images.githubusercontent.com/1818302/90414618-0570c880-e07e-11ea-8fb5-f261de4b4506.png">

It is 115mb zipped and available at `s3://usfimr-data/shp/USFIMR_merged.zip`.

PR incoming in a bit with build_catalog.sh updated to pull this file instead after I've verified we've generated a correct catalog.

**Comment by CloudNiner** [comment]
Edit: The above didn't work -- for some reason when I merge via ogr2ogr or ogrmerge.py, the list of features in the merged shapefile somehow has 400k entries even though each individual input shapefile only has one large polygon feature. 

PR incoming with some additional changes to pull, reproject and directly generate items from each of the individual shapefiles.

**Comment by CloudNiner** [comment]
Edit 2: What a whirlwind. With the removal of the GloFIMR Bangladesh flood shapefile that was a part of the individual file list, I was able to correctly manually merge the remaining files together with a consistent attribute table and 1 feature per shapefile.

Looks like the Bangladesh file was added erroneously as an experiment, its part of GloFIMR instead of USFIMR so it makes sense to remove from our merged shapefile.
